doc/whats_new/v0.22.rst

```diff
@@ -173,6 +173,12 @@ Changelog
   maximum number of function evaluation to not meet ``tol`` improvement.
   :issue:`9274` by :user:`Daniel Perry <daniel-perry>`.
 
+:mod:`sklearn.cross_decomposition`
+..................................
+
+- |Fix| Fixed a bug where :func:`cross_decomposition._pls.fit` was raising an
```

Comment:
The naming of `_pls` implies private API. Please reference public API that is affected by this change instead